zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -44,6 +43,12 @@
 POSTGRESQL_DEFAULT_PORT = 5432
 POSTGRESQL_CONNECT_TIMEOUT = os.environ.get('ZMON_AGENT_POSTGRESQL_CONNECT_TIMEOUT', 2)
 
+HPA_TYPE = 'kube_hpa'
+
+# Custom Resources
+VPA_TYPE = 'kube_vpa'
```

Comment:
This is not implemented?

Reply:
It's not. I can remove it or implement it. I think as we should change the way these things are exposed. I'm not sure if exposing all these things as entities is the best idea. But maybe we can tackle that later.

Reply:
Maybe it's best to just leave it out of this PR IMO.

Reply:
Done